stable/weave-scope/charts/weave-scope-agent/templates/clusterrole.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if .Values.rbac }}
```

Comment:
The outer if statement is unnecessary.

Reply:
I have a similar structure in the daemonset template:

```
      {{- if .Values.rbac }}
        {{- if .Values.rbac.create }}
      serviceAccountName: {{ required "rbac.serviceAccountName must be set if rbac.create is true" .Values.rbac.serviceAccountName }}
        {{- else if .Values.rbac.serviceAccountName }}
      serviceAccountName: {{ .Values.rbac.serviceAccountName }}
        {{- end }}
{{- end }}
```

Since the only difference in the behavior of the template between `rbac.create: true` and `rbac.create: false` is putting a "required" on the ServiceAccount, and there's already a value for that set in values.yaml, can I just collapse that whole block back down to `serviceAccountName: {{ .Values.rbac.serviceAccountName }}` then?

Reply:
yes